nm: Fix dbus value type of coalesce-adaptive RX/TX

cathay4t opened the pull request:
When modifying `adaptive-rx` of `coalesce` in ethtool, we will get failure:

    org.freedesktop.NetworkManager.Settings.Connection.InvalidProperty:
    ethtool.coalesce-adaptive-rx: setting has invalid variant type

The `coalesce-adaptive-rx` and `coalesce-adaptive-tx` in NM dbus is u32,
other than bool.

Special hardware required to test this, hence the integration test case
has verification disabled, but it still check the communication to
NetworkManager.

ffmancera commented on tests/integration/nm/ethtool_test.py:
```diff
@@ -0,0 +1,38 @@
+## SPDX-License-Identifier: LGPL-2.1-or-later
```
The python linter is complaining.. please use a single `#`. Thanks!

cathay4t replied:
Resolved. Thanks.

cathay4t committed:
nm: Fix dbus value type of coalesce-adaptive RX/TX

When modifying `adaptive-rx` of `coalesce` in ethtool, we will get failure:

    org.freedesktop.NetworkManager.Settings.Connection.InvalidProperty:
    ethtool.coalesce-adaptive-rx: setting has invalid variant type

The `coalesce-adaptive-rx` and `coalesce-adaptive-tx` in NM dbus is u32,
other than bool.

Special hardware required to test this, hence the integration test case
has verification disabled, but it still check the communication to
NetworkManager.

Signed-off-by: Gris Ge <fge@redhat.com>
